library/cohesity_job.py

```diff
@@ -381,17 +381,46 @@ def parse_vmware_protection_sources_json(response, vm_names):
     return list(set(ids))
 
 
+def get_tag_ids(module, tags, parentSourceId, token):
```

Comment:
use underscores if helper functions, also write the type and describe the params , since this is public code

Reply:
done